libultraship/libultraship/Lib/Fast3D/gfx_sdl2.cpp

```diff
@@ -15,13 +15,18 @@
 #include "SDL_opengl.h"
 #elif __APPLE__
 #include <SDL.h>
+#elif __SWITCH__
+#include <SDL2/SDL.h>
+#include <switch.h>
+#include "glad/glad.h"
```

Comment:
we're importing a lib here 
```suggestion
#include <glad/glad.h>
```